Error creating EKS Node Group when   manage_cluster_iam_resources set to false: ResourceNotFoundException: No cluster found for name

# I have issues
## I'm submitting a...

* [ ] bug report
* [ ] feature request
* [X] support request - read the [FAQ](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/faq.md) first!
* [ ] kudos, thank you, warm fuzzy

## What is the current behavior?
Only happens when manage_cluster_iam_resources = false. Cluster has been created but not Node Group. Getting error showing below. And the service account I am using has not right. That's why.
If manage_cluster_iam_resources = false there is no problem at all. 

```log
Error: error creating EKS Node Group (kobeeks-module-20:kobeeks-module-20-worker-node-evolving-anteater): ResourceNotFoundException: No cluster found for name: kobeeks-module-20.
{
  Message_: "No cluster found for name: kobeeks-module-20."
}
```
## If this is a bug, how to reproduce? Please include a code sample if relevant.

apply the main.tf shows below

```tf
terraform {
  required_version = ">= 0.12.6"
}

provider "aws" {
  version = ">= 2.28.1"
  region  = var.region
}

provider "random" {
  version = "~> 2.1"
}

provider "local" {
  version = "~> 1.2"
}

provider "null" {
  version = "~> 2.1"
}

provider "template" {
  version = "~> 2.1"
}

module "eks" {
  source       = "../.."
  cluster_name = var.cluster_name
  subnets      = var.subnet_ids

  # create_eks = false

  tags = {
    Environment = var.environment
    Description  = var.description
    ApplicationId = var.applicationid
    Transit = var.transit
  }
  
  manage_aws_auth = false
  
  manage_cluster_iam_resources = false
  cluster_create_security_group = false
  cluster_iam_role_name = var.cluster_service_role

  manage_worker_iam_resources = false
  worker_create_security_group = false
  
  cluster_version = var.cluster_version
  # cluster_endpoint_private_access = var.cluster_endpoint_private_access
  # cluster_endpoint_public_access = var.cluster_endpoint_public_access

  vpc_id = var.vpc_id

  node_groups_defaults = {
    ami_type  = "AL2_x86_64"
    disk_size = 50
  }

  node_groups = {
    worker-node = {
      iam_role_arn = "arn:aws:iam::XXXXXXXXXXXXXXXXX:role/EKSNodeGroupServiceRole"
      desired_capacity = var.autoscaling_desired_size
      max_capacity     = var.autoscaling_max_size
      min_capacity     = var.autoscaling_min_size

      instance_type = var.instance_type
      k8s_labels = {
        Environment = var.environment
        Description  = var.description
        ApplicationId = var.applicationid
        Transit = var.transit
      }
      additional_tags = {
        ExtraTag = "worker-node"
      }
    }
  }
}
```

## What's the expected behavior?
Node group create following the cluster.


## Are you able to fix this problem and submit a PR? Link here if you have already.
Tried hard. But not able to.

## Environment details

* Affected module version: 10.0
* OS: CentOS
* Terraform version: v0.12.19

## Any other relevant info

Terraform Plan:

```bash
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

module.eks.data.aws_ami.eks_worker_windows: Refreshing state...
module.eks.data.aws_iam_policy_document.cluster_assume_role_policy: Refreshing state...
module.eks.data.aws_region.current: Refreshing state...
module.eks.data.aws_iam_role.custom_cluster_iam_role[0]: Refreshing state...
module.eks.data.aws_ami.eks_worker: Refreshing state...
module.eks.data.aws_caller_identity.current: Refreshing state...
module.eks.data.aws_iam_policy_document.workers_assume_role_policy: Refreshing state...

------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create
 <= read (data resources)

Terraform will perform the following actions:

  # module.eks.data.null_data_source.node_groups[0] will be read during apply
  # (config refers to values not yet known)
 <= data "null_data_source" "node_groups"  {
      + has_computed_default = (known after apply)
      + id                   = (known after apply)
      + inputs               = {
          + "aws_auth"        = ""
          + "cluster_name"    = "kobeeks-module-20"
          + "role_CNI_Policy" = ""
          + "role_Container"  = ""
          + "role_NodePolicy" = ""
        }
      + outputs              = (known after apply)
      + random               = (known after apply)
    }

  # module.eks.data.template_file.kubeconfig[0] will be read during apply
  # (config refers to values not yet known)
 <= data "template_file" "kubeconfig"  {
      + id       = (known after apply)
      + rendered = (known after apply)
      + template = <<~EOT
            apiVersion: v1
            preferences: {}
            kind: Config
            
            clusters:
            - cluster:
                server: ${endpoint}
                certificate-authority-data: ${cluster_auth_base64}
              name: ${kubeconfig_name}
            
            contexts:
            - context:
                cluster: ${kubeconfig_name}
                user: ${kubeconfig_name}
              name: ${kubeconfig_name}
            
            current-context: ${kubeconfig_name}
            
            users:
            - name: ${kubeconfig_name}
              user:
                exec:
                  apiVersion: client.authentication.k8s.io/v1alpha1
                  command: ${aws_authenticator_command}
                  args:
            ${aws_authenticator_command_args}
            ${aws_authenticator_additional_args}
            ${aws_authenticator_env_variables}
        EOT
      + vars     = {
          + "aws_authenticator_additional_args" = ""
          + "aws_authenticator_command"         = "aws-iam-authenticator"
          + "aws_authenticator_command_args"    = <<~EOT
                        - "token"
                        - "-i"
                        - "kobeeks-module-20"
            EOT
          + "aws_authenticator_env_variables"   = ""
          + "cluster_auth_base64"               = (known after apply)
          + "endpoint"                          = (known after apply)
          + "kubeconfig_name"                   = "eks_kobeeks-module-20"
        }
    }

  # module.eks.data.template_file.node_group_arns[0] will be read during apply
  # (config refers to values not yet known)
 <= data "template_file" "node_group_arns"  {
      + id       = (known after apply)
      + rendered = (known after apply)
      + template = <<~EOT
            - rolearn: ${worker_role_arn}
              username: system:node:{{EC2PrivateDNSName}}
              groups:
                - system:bootstrappers
                - system:nodes
                %{~ if platform == "windows" ~}
                - eks:kube-proxy-windows
                %{~ endif ~}
        EOT
      + vars     = {
          + "platform"        = "linux"
          + "worker_role_arn" = "arn:aws:iam::xxxxxxxxxxxxxxxxx:role/EKSNodeGroupServiceRole"
        }
    }

  # module.eks.aws_eks_cluster.this[0] will be created
  + resource "aws_eks_cluster" "this" {
      + arn                   = (known after apply)
      + certificate_authority = (known after apply)
      + created_at            = (known after apply)
      + endpoint              = (known after apply)
      + id                    = (known after apply)
      + identity              = (known after apply)
      + name                  = "kobeeks-module-20"
      + platform_version      = (known after apply)
      + role_arn              = "arn:aws:iam::xxxxxxxxxxxxxxxx:role/EKSClusterServiceRole"
      + status                = (known after apply)
      + tags                  = {
          + "ApplicationId" = "5482"
          + "Description"   = "eks"
          + "Environment"   = "dev"
          + "Transit"       = "xxxxxxxxxxxx"
        }
      + version               = "1.14"

      + timeouts {
          + create = "30m"
          + delete = "15m"
        }

      + vpc_config {
          + cluster_security_group_id = (known after apply)
          + endpoint_private_access   = false
          + endpoint_public_access    = true
          + public_access_cidrs       = [
              + "0.0.0.0/0",
            ]
          + security_group_ids        = [
              + "",
            ]
          + subnet_ids                = [
              + "subnet-xxxxxxxxxxxxxxxxxxx",
              + "subnet-xxxxxxxxxxxxxxxxxxx",
            ]
          + vpc_id                    = (known after apply)
        }
    }

  # module.eks.local_file.kubeconfig[0] will be created
  + resource "local_file" "kubeconfig" {
      + content              = (known after apply)
      + directory_permission = "0777"
      + file_permission      = "0777"
      + filename             = "./kubeconfig_kobeeks-module-20"
      + id                   = (known after apply)
    }

  # module.eks.module.node_groups.aws_eks_node_group.workers["worker-node"] will be created
  + resource "aws_eks_node_group" "workers" {
      + ami_type        = "AL2_x86_64"
      + arn             = (known after apply)
      + cluster_name    = (known after apply)
      + disk_size       = 50
      + id              = (known after apply)
      + instance_types  = [
          + "t3.medium",
        ]
      + labels          = {
          + "ApplicationId" = "5482"
          + "Description"   = "eks"
          + "Environment"   = "dev"
          + "Transit"       = "xxxxxx"
        }
      + node_group_name = (known after apply)
      + node_role_arn   = "arn:aws:iam::xxxxxxxxx:role/EKSNodeGroupServiceRole"
      + release_version = (known after apply)
      + resources       = (known after apply)
      + status          = (known after apply)
      + subnet_ids      = [
          + "subnet-xxxxxxxxxxxxxxxxxxxx",
          + "subnet-xxxxxxxxxxxxxxxxxxxx",
        ]
      + tags            = {
          + "ApplicationId" = "5482"
          + "Description"   = "eks"
          + "Environment"   = "dev"
          + "ExtraTag"      = "worker-node"
          + "Transit"       = "xxxxx"
        }
      + version         = (known after apply)

      + scaling_config {
          + desired_size = 2
          + max_size     = 6
          + min_size     = 2
        }
    }

  # module.eks.module.node_groups.random_pet.node_groups["worker-node"] will be created
  + resource "random_pet" "node_groups" {
      + id        = (known after apply)
      + keepers   = (known after apply)
      + length    = 2
      + separator = "-"
    }

Plan: 4 to add, 0 to change, 0 to destroy.
```

- Part of terraform.log 

```log
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:40 [DEBUG] [aws-sdk-go] DEBUG: Response sts/GetCallerIdentity Details:
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ RESPONSE ]--------------------------------------
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: HTTP/1.1 200 OK
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Connection: close
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 461
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: text/xml
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Date: Fri, 13 Mar 2020 21:59:40 GMT
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Requestid: 2e360512-6cbd-4d78-b17d-59e841e5b151
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:40 [DEBUG] [aws-sdk-go] <GetCallerIdentityResponse xmlns="https://sts.amazonaws.com/doc/2011-06-15/">
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   <GetCallerIdentityResult>
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     <Arn>arn:aws:iam::xxxxxxxxxxxxxx:user/sa-0242744960f49b0c90e8318ab74720a13f4cdf7047fbba961e12551b55273</Arn>
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     <UserId>AIDAQIUNC3ZD6DZX4Z5EC</UserId>
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     <Account>xxxxxxxxxxxxxx</Account>
</GetCallerIdentityResult>
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   <ResponseMetadata>
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     <RequestId>2e360512-6cbd-4d78-b17d-59e841e5b151</RequestId>
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   </ResponseMetadata>
2020-03-13T17:59:40.943-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: </GetCallerIdentityResponse>
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:40 [DEBUG] [aws-sdk-go] DEBUG: Request ec2/DescribeAccountAttributes Details:
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ REQUEST POST-SIGN ]-----------------------------
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: POST / HTTP/1.1
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Host: ec2.ca-central-1.amazonaws.com
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: User-Agent: aws-sdk-go/1.29.20 (go1.13.7; linux; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.12.19 (+https://www.terraform.io)
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 87
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Authorization: AWS4-HMAC-SHA256 Credential=xxxxxxxxxxxxxxxxxxxx/20200313/ca-central-1/ec2/aws4_request, SignedHeaders=content-length;content-type;host;x-amz-date, Signature=76e5cf2dd8f0acd1acf5f12b186a14b103c90bbd5e57802b3b7eea4d7d9b8920
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/x-www-form-urlencoded; charset=utf-8
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Date: 20200313T215940Z
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Accept-Encoding: gzip
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Action=DescribeAccountAttributes&AttributeName.1=supported-platforms&Version=2016-11-15
2020-03-13T17:59:40.946-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] [aws-sdk-go] DEBUG: Response ec2/DescribeAccountAttributes Details:
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ RESPONSE ]--------------------------------------
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: HTTP/1.1 200 OK
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Connection: close
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 540
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: text/xml;charset=UTF-8
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Date: Fri, 13 Mar 2020 21:59:40 GMT
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Server: AmazonEC2
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] [aws-sdk-go] <?xml version="1.0" encoding="UTF-8"?>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: <DescribeAccountAttributesResponse xmlns="http://ec2.amazonaws.com/doc/2016-11-15/">
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     <requestId>543bb070-f1fd-4bfa-a651-2b7e57e28573</requestId>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     <accountAttributeSet>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         <item>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:             <attributeName>supported-platforms</attributeName>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:             <attributeValueSet>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:                 <item>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:                     <attributeValue>VPC</attributeValue>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:                 </item>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:             </attributeValueSet>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         </item>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     </accountAttributeSet>
2020-03-13T17:59:41.232-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: </DescribeAccountAttributesResponse>
2020/03/13 17:59:41 [WARN] Provider "registry.terraform.io/-/aws" produced an invalid plan for module.eks.aws_eks_cluster.this[0], but we are tolerating it because it is using the legacy plugin SDK.
    The following problems may be the cause of any confusing errors from downstream operations:
      - .enabled_cluster_log_types: planned value cty.NullVal(cty.Set(cty.String)) does not match config value cty.SetValEmpty(cty.String)
2020/03/13 17:59:41 [DEBUG] module.eks.aws_eks_cluster.this[0]: applying the planned Create change
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] setting computed for "certificate_authority" from ComputedKeys
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] setting computed for "identity" from ComputedKeys
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] Creating EKS Cluster: {
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   Logging: {
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     ClusterLogging: [{
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         Enabled: true,
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         Types: []
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       },{
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         Enabled: false,
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         Types: [
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:           "controllerManager",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:           "scheduler",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:           "authenticator",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:           "audit",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:           "api"
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         ]
s_v2.53.0_x4:       }]
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   },
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   Name: "kobeeks-module-20",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   ResourcesVpcConfig: {
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     EndpointPrivateAccess: false,
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     EndpointPublicAccess: true,
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     PublicAccessCidrs: ["0.0.0.0/0"],
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     SecurityGroupIds: [],
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     SubnetIds: ["subnet-xxxxxxxxxxxxxxxxxxxx","subnet-xxxxxxxxxxxxxxxxxxxx"]
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   },
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   RoleArn: "arn:aws:iam::xxxxxxxxxxxxxx:role/EKSClusterServiceRole",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   Tags: {
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     Description: "eks",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     Environment: "dev",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     Transit: "73901",
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     ApplicationId: "5482"
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   },
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   Version: "1.14"
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: }
ess]
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] [aws-sdk-go] DEBUG: Request eks/CreateCluster Details:
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ REQUEST POST-SIGN ]-----------------------------
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: POST /clusters HTTP/1.1
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Host: eks.ca-central-1.amazonaws.com
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: User-Agent: aws-sdk-go/1.29.20 (go1.13.7; linux; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.12.19 (+https://www.terraform.io)
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 614
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Authorization: AWS4-HMAC-SHA256 Credential=xxxxxxxxxxxxxxxxxxxx/20200313/ca-central-1/eks/aws4_request, SignedHeaders=content-length;content-type;host;x-amz-date, Signature=fd61de65b370b07c35afb001449eabf1003bb81fa2766958319ce3831b293ed1
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Date: 20200313T215941Z
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Accept-Encoding: gzip
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
a907c1"]},"roleArn":"arn:aws:iam::xxxxxxxxxxxxxx:role/EKSClusterServiceRole","tags":{"ApplicationId":"5482","Description":"eks","Environment":"dev","Transit":"73901"},"version":"1.14"}
2020-03-13T17:59:41.249-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:41.255-0400 [DEBUG] plugin: plugin process exited: path=/home/p17m@res.bngf.local/git/bb/eks/terraform/.terraform/plugins/linux_amd64/terraform-provider-null_v2.1.2_x4 pid=10005
2020-03-13T17:59:41.255-0400 [DEBUG] plugin: plugin exited
2020/03/13 17:59:41 [DEBUG] module.eks.module.node_groups.random_pet.node_groups["worker-node"]: applying the planned Create change
2020-03-13T17:59:41.264-0400 [DEBUG] plugin: plugin process exited: path=/home/p17m@res.bngf.local/git/bb/eks/terraform/.terraform/plugins/linux_amd64/terraform-provider-random_v2.2.1_x4 pid=10015
2020-03-13T17:59:41.264-0400 [DEBUG] plugin: plugin exited
2020/03/13 17:59:41 [DEBUG] module.eks.module.node_groups.aws_eks_node_group.workers["worker-node"]: applying the planned Create change
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] setting computed for "resources" from ComputedKeys
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] [aws-sdk-go] DEBUG: Request eks/CreateNodegroup Details:
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ REQUEST POST-SIGN ]-----------------------------
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: POST /clusters/kobeeks-module-20/node-groups HTTP/1.1
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Host: eks.ca-central-1.amazonaws.com
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: User-Agent: aws-sdk-go/1.29.20 (go1.13.7; linux; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.12.19 (+https://www.terraform.io)
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 593
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Authorization: AWS4-HMAC-SHA256 Credential=xxxxxxxxxxxxxxxxxxxx/20200313/ca-central-1/eks/aws4_request, SignedHeaders=content-length;content-type;host;x-amz-date, Signature=c5bd4156a381e0fe207084ca0a4f641295d0b89de61d91eaebc9d5384d5814f0
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Date: 20200313T215941Z
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Accept-Encoding: gzip
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: {"amiType":"AL2_x86_64","clientRequestToken":"terraform-20200313215941273000000001","diskSize":50,"instanceTypes":["t3.medium"],"labels":{"ApplicationId":"5482","Description":"eks","Environment":"dev","Transit":"73901"},"nodeRole":"arn:aws:iam::xxxxxxxxxxxxxx:role/EKSNodeGroupServiceRole","nodegroupName":"kobeeks-module-20-worker-node-evolving-anteater","scalingConfig":{"desiredSize":2,"maxSize":6,"minSize":2},"subnets":["subnet-xxxxxxxxxxxxxxxxxxxx","subnet-xxxxxxxxxxxxxxxxxxxx"],"tags":{"ApplicationId":"5482","Description":"eks","Environment":"dev","ExtraTag":"worker-node","Transit":"73901"}}
2020-03-13T17:59:41.273-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] [aws-sdk-go] DEBUG: Response eks/CreateNodegroup Details:
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ RESPONSE ]--------------------------------------
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: HTTP/1.1 404 Not Found
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Connection: close
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 146
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Date: Fri, 13 Mar 2020 21:59:41 GMT
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Apigw-Id: JWVMIEmxYosFjZA=
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Errortype: ResourceNotFoundException
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Requestid: 2e054a3f-bbc1-40b0-8530-cf25081ed3eb
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Trace-Id: Root=1-5e6c024d-b7fc9db559bcc5453752ce0c
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] [aws-sdk-go] {
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   "clusterName" : null,
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   "nodegroupName" : null,
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   "fargateProfileName" : null,
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   "message" : "No cluster found for name: kobeeks-module-20."
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: }
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:41 [DEBUG] [aws-sdk-go] DEBUG: Validate Response eks/CreateNodegroup failed, attempt 0/25, error ResourceNotFoundException: No cluster found for name: kobeeks-module-20.
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: {
s-module-20."
2020-03-13T17:59:41.736-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: }
2020/03/13 17:59:41 [DEBUG] module.eks.module.node_groups.aws_eks_node_group.workers["worker-node"]: apply errored, but we're indicating that via the Error pointer rather than returning it: error creating EKS Node Group (kobeeks-module-20:kobeeks-module-20-worker-node-evolving-anteater): ResourceNotFoundException: No cluster found for name: kobeeks-module-20.
{
  Message_: "No cluster found for name: kobeeks-module-20."
}
2020/03/13 17:59:41 [ERROR] module.eks.module.node_groups: eval: *terraform.EvalApplyPost, err: error creating EKS Node Group (kobeeks-module-20:kobeeks-module-20-worker-node-evolving-anteater): ResourceNotFoundException: No cluster found for name: kobeeks-module-20.
{
  Message_: "No cluster found for name: kobeeks-module-20."
}
2020/03/13 17:59:41 [ERROR] module.eks.module.node_groups: eval: *terraform.EvalSequence, err: error creating EKS Node Group (kobeeks-module-20:kobeeks-module-20-worker-node-evolving-anteater): ResourceNotFoundException: No cluster found for name: kobeeks-module-20.
{
  Message_: "No cluster found for name: kobeeks-module-20."
}
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:42 [DEBUG] [aws-sdk-go] DEBUG: Response eks/CreateCluster Details:
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ RESPONSE ]--------------------------------------
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: HTTP/1.1 200 OK
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Connection: close
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 1160
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Date: Fri, 13 Mar 2020 21:59:42 GMT
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Apigw-Id: JWVMIHcKYosFWkQ=
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Requestid: a88c9747-472f-40b6-af89-7ea9b19a6290
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Trace-Id: Root=1-5e6c024d-dca5b980a2908380f533b840
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:42 [DEBUG] [aws-sdk-go] {
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   "cluster" : {
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "name" : "kobeeks-module-20",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "arn" : "arn:aws:eks:ca-central-1:xxxxxxxxxxxxxx:cluster/kobeeks-module-20",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "createdAt" : 1.584136782209E9,
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "version" : "1.14",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "endpoint" : null,
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "roleArn" : "arn:aws:iam::xxxxxxxxxxxxxx:role/EKSClusterServiceRole",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "resourcesVpcConfig" : {
     "subnetIds" : [ "subnet-xxxxxxxxxxxxxxxxxxxx", "subnet-xxxxxxxxxxxxxxxxxxxx" ],
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "securityGroupIds" : [ ],
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "clusterSecurityGroupId" : null,
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "vpcId" : "vpc-xxxxxxxxxxxxxxxxxxxx",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "endpointPublicAccess" : true,
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "endpointPrivateAccess" : false,
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "publicAccessCidrs" : [ "0.0.0.0/0" ]
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "logging" : {
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "clusterLogging" : [ {
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         "types" : [ "api", "audit", "authenticator", "controllerManager", "scheduler" ],
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         "enabled" : false
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       } ]
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "identity" : null,
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "status" : "CREATING",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "certificateAuthority" : {
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "data" : null
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "clientRequestToken" : null,
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "platformVersion" : "eks.9",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "tags" : {
17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Environment" : "dev",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Description" : "eks",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "ApplicationId" : "5482",
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Transit" : "73901"
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "encryptionConfig" : null
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   }
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: }
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:42 [DEBUG] Waiting for state to become: [ACTIVE]
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:42 [DEBUG] [aws-sdk-go] DEBUG: Request eks/DescribeCluster Details:
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ REQUEST POST-SIGN ]-----------------------------
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: GET /clusters/kobeeks-module-20 HTTP/1.1
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Host: eks.ca-central-1.amazonaws.com
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: User-Agent: aws-sdk-go/1.29.20 (go1.13.7; linux; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.12.19 (+https://www.terraform.io)
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Authorization: AWS4-HMAC-SHA256 Credential=xxxxxxxxxxxxxxxxxxxx/20200313/ca-central-1/eks/aws4_request, SignedHeaders=content-type;host;x-amz-date, Signature=4b8a01fff6dafa07e229a584d2a48d3282f0744531a479bf82c699cc6b71d851
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Date: 20200313T215942Z
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Accept-Encoding: gzip
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:42.308-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:42 [DEBUG] [aws-sdk-go] DEBUG: Response eks/DescribeCluster Details:
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ RESPONSE ]--------------------------------------
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: HTTP/1.1 200 OK
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Connection: close
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 1160
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Date: Fri, 13 Mar 2020 21:59:42 GMT
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Apigw-Id: JWVMSFZ34osFoKQ=
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Requestid: 7a7b8638-a692-41b1-a31e-063461459653
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Trace-Id: Root=1-5e6c024e-e5118e264a7ea614efae6e6e
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:42 [DEBUG] [aws-sdk-go] {
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   "cluster" : {
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "name" : "kobeeks-module-20",
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "arn" : "arn:aws:eks:ca-central-1:xxxxxxxxxxxxxx:cluster/kobeeks-module-20",
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "createdAt" : 1.584136782209E9,
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "version" : "1.14",
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "endpoint" : null,
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "roleArn" : "arn:aws:iam::xxxxxxxxxxxxxx:role/EKSClusterServiceRole",
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "resourcesVpcConfig" : {
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "subnetIds" : [ "subnet-xxxxxxxxxxxxxxxxxxxx", "subnet-xxxxxxxxxxxxxxxxxxxx" ],
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "securityGroupIds" : [ ],
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "clusterSecurityGroupId" : null,
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "vpcId" : "vpc-xxxxxxxxxxxxxxxxxxxx",
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "endpointPublicAccess" : true,
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "endpointPrivateAccess" : false,
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "publicAccessCidrs" : [ "0.0.0.0/0" ]
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "logging" : {
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "clusterLogging" : [ {
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         "types" : [ "api", "audit", "authenticator", "controllerManager", "scheduler" ],
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         "enabled" : false
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       } ]
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:42.613-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "identity" : null,
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "status" : "CREATING",
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "certificateAuthority" : {
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "data" : null
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "clientRequestToken" : null,
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "platformVersion" : "eks.9",
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "tags" : {
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Environment" : "dev",
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Description" : "eks",
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "ApplicationId" : "5482",
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Transit" : "73901"
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "encryptionConfig" : null
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   }
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: }
2020-03-13T17:59:42.614-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:42 [TRACE] Waiting 200ms before next try
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:42 [DEBUG] [aws-sdk-go] DEBUG: Request eks/DescribeCluster Details:
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ REQUEST POST-SIGN ]-----------------------------
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: GET /clusters/kobeeks-module-20 HTTP/1.1
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Host: eks.ca-central-1.amazonaws.com
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: User-Agent: aws-sdk-go/1.29.20 (go1.13.7; linux; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.12.19 (+https://www.terraform.io)
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Authorization: AWS4-HMAC-SHA256 Credential=xxxxxxxxxxxxxxxxxxxx/20200313/ca-central-1/eks/aws4_request, SignedHeaders=content-type;host;x-amz-date, Signature=4b8a01fff6dafa07e229a584d2a48d3282f0744531a479bf82c699cc6b71d851
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Date: 20200313T215942Z
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Accept-Encoding: gzip
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:42.814-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:43 [DEBUG] [aws-sdk-go] DEBUG: Response eks/DescribeCluster Details:
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ RESPONSE ]--------------------------------------
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: HTTP/1.1 200 OK
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Connection: close
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 1160
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Date: Fri, 13 Mar 2020 21:59:43 GMT
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Apigw-Id: JWVMXG3B4osFQHw=
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Requestid: 9ba07252-bc94-48c7-9a2e-bdfb01858591
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Trace-Id: Root=1-5e6c024f-a3ba31842ca7f94ab978cb92
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:43 [DEBUG] [aws-sdk-go] {
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   "cluster" : {
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "name" : "kobeeks-module-20",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "arn" : "arn:aws:eks:ca-central-1:xxxxxxxxxxxxxx:cluster/kobeeks-module-20",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "createdAt" : 1.584136782209E9,
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "version" : "1.14",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "endpoint" : null,
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "roleArn" : "arn:aws:iam::xxxxxxxxxxxxxx:role/EKSClusterServiceRole",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "resourcesVpcConfig" : {
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "subnetIds" : [ "subnet-xxxxxxxxxxxxxxxxxxxx", "subnet-xxxxxxxxxxxxxxxxxxxx" ],
EBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "securityGroupIds" : [ ],
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "clusterSecurityGroupId" : null,
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "vpcId" : "vpc-xxxxxxxxxxxxxxxxxxxx",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "endpointPublicAccess" : true,
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "endpointPrivateAccess" : false,
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "publicAccessCidrs" : [ "0.0.0.0/0" ]
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "logging" : {
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "clusterLogging" : [ {
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         "types" : [ "api", "audit", "authenticator", "controllerManager", "scheduler" ],
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:         "enabled" : false
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       } ]
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "identity" : null,
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "status" : "CREATING",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "certificateAuthority" : {
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "data" : null
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "clientRequestToken" : null,
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "platformVersion" : "eks.9",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "tags" : {
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Environment" : "dev",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Description" : "eks",
EBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "ApplicationId" : "5482",
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:       "Transit" : "73901"
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     },
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:     "encryptionConfig" : null
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4:   }
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: }
2020-03-13T17:59:43.121-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:43 [TRACE] Waiting 400ms before next try
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:43 [DEBUG] [aws-sdk-go] DEBUG: Request eks/DescribeCluster Details:
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ REQUEST POST-SIGN ]-----------------------------
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: GET /clusters/kobeeks-module-20 HTTP/1.1
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Host: eks.ca-central-1.amazonaws.com
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: User-Agent: aws-sdk-go/1.29.20 (go1.13.7; linux; amd64) APN/1.0 HashiCorp/1.0 Terraform/0.12.19 (+https://www.terraform.io)
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Authorization: AWS4-HMAC-SHA256 Credential=xxxxxxxxxxxxxxxxxxxx/20200313/ca-central-1/eks/aws4_request, SignedHeaders=content-type;host;x-amz-date, Signature=8675846e2996a85fbdff59e68bae2199df93c85eecaca55d914ed54075d2cda8
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Date: 20200313T215943Z
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Accept-Encoding: gzip
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:43.522-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 2020/03/13 17:59:43 [DEBUG] [aws-sdk-go] DEBUG: Response eks/DescribeCluster Details:
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: ---[ RESPONSE ]--------------------------------------
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: HTTP/1.1 200 OK
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Connection: close
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Length: 1160
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Content-Type: application/json
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: Date: Fri, 13 Mar 2020 21:59:43 GMT
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amz-Apigw-Id: JWVMeFTx4osFlSg=
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Requestid: 758eacf7-f868-4764-a22b-0d5ca014b46c
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: X-Amzn-Trace-Id: Root=1-5e6c024f-e2de7780a64294006ce32980
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: 
2020-03-13T17:59:43.851-0400 [DEBUG] plugin.terraform-provider-aws_v2.53.0_x4: -----------------------------------------------------
```

--- a/node_groups.tf
+++ b/node_groups.tf
@@ -4,7 +4,7 @@ data "null_data_source" "node_groups" {
   count = var.create_eks ? 1 : 0
 
   inputs = {
-    cluster_name = var.cluster_name
+    cluster_name = aws_eks_cluster.this[0].name
 
     # Ensure these resources are created before "unlocking" the data source.
     # `depends_on` causes a refresh on every run so is useless here.

